<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="appearance_tab">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="clock_group">
         <property name="title">
          <string>Clock</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="2">
           <layout class="QGridLayout" name="sliders_layout">
            <item row="0" column="0">
             <widget class="QLabel" name="opacity_label">
              <property name="text">
               <string>opacity</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="opacity_slider">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="sliderPosition">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="opacity_value">
              <property name="text">
               <string notr="true">100</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="opcacity_percent">
              <property name="text">
               <string notr="true">%</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="zoom_label">
              <property name="text">
               <string>zoom</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="zoom_slider">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>400</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="zoom_value">
              <property name="text">
               <string notr="true">100</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="zoom_percent">
              <property name="text">
               <string notr="true">%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QCheckBox" name="use_customization">
            <property name="text">
             <string>use skin customization (change color or apply texture)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="checkbox_layout">
            <item>
             <widget class="QCheckBox" name="stay_on_top">
              <property name="text">
               <string>stay on top</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="transp_for_input">
              <property name="text">
               <string>transparent for input</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="sep_flash">
              <property name="text">
               <string>separator flashes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="skin_group">
         <property name="title">
          <string>Skin</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
            <item>
             <widget class="QRadioButton" name="use_skin">
              <property name="text">
               <string>use skin</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="skin_box">
              <property name="minimumSize">
               <size>
                <width>155</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
            <item row="0" column="0">
             <widget class="QLabel" name="version_label">
              <property name="text">
               <string>Version :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="version_value">
              <property name="text">
               <string>unknown</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="author_value">
              <property name="text">
               <string>unknown</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="email_label">
              <property name="text">
               <string>E-mail :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="author_label">
              <property name="text">
               <string>Author :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="email_value">
              <property name="text">
               <string>unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="use_font">
              <property name="text">
               <string>use font</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="sel_font_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="texture_group">
         <property name="title">
          <string>Texture</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QGroupBox" name="txd_type_group">
            <property name="title">
             <string>Type</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QRadioButton" name="type_color">
               <property name="text">
                <string>solid color</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="type_image">
               <property name="text">
                <string>custom image</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="sel_image_btn">
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="sel_color_btn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="mode_group">
            <property name="title">
             <string>Mode</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="mode_stretch">
               <property name="text">
                <string>stretch</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="mode_tile">
               <property name="text">
                <string>tile</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="txd_per_elem">
            <property name="text">
             <string>texture per element</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="misc_tab">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="time_format_group">
         <property name="title">
          <string>Time format</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="system_format">
            <property name="text">
             <string>system</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="custom_format">
            <property name="text">
             <string>custom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="format_box">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
            <item>
             <property name="text">
              <string notr="true">h:mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">hh:mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">hh:mm:ss</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">h:mmA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">hh:mmA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">hh:mm:ssA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">H:mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">HH:mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">HH:mm:ss</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="apply_btn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="autoupdate_group">
         <property name="title">
          <string>Autoupdate</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="enable_autoupdate">
            <property name="text">
             <string>enable autoupdate</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="update_period_label">
            <property name="text">
             <string>Check for updates every</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="update_period_box"/>
          </item>
          <item>
           <widget class="QCheckBox" name="check_for_beta">
            <property name="text">
             <string>check for beta releases</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>225</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="plugins_tab">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="plugins_list"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacity_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacity_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoom_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zoom_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>type_color</sender>
   <signal>toggled(bool)</signal>
   <receiver>sel_color_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>type_image</sender>
   <signal>toggled(bool)</signal>
   <receiver>sel_image_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_skin</sender>
   <signal>toggled(bool)</signal>
   <receiver>skin_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_font</sender>
   <signal>toggled(bool)</signal>
   <receiver>sel_font_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_skin</sender>
   <signal>toggled(bool)</signal>
   <receiver>version_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_skin</sender>
   <signal>toggled(bool)</signal>
   <receiver>author_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_skin</sender>
   <signal>toggled(bool)</signal>
   <receiver>email_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_skin</sender>
   <signal>toggled(bool)</signal>
   <receiver>version_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_skin</sender>
   <signal>toggled(bool)</signal>
   <receiver>author_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_skin</sender>
   <signal>toggled(bool)</signal>
   <receiver>email_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_customization</sender>
   <signal>toggled(bool)</signal>
   <receiver>texture_group</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_font</sender>
   <signal>toggled(bool)</signal>
   <receiver>use_customization</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_format</sender>
   <signal>toggled(bool)</signal>
   <receiver>format_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_format</sender>
   <signal>toggled(bool)</signal>
   <receiver>apply_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_format</sender>
   <signal>clicked()</signal>
   <receiver>apply_btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enable_autoupdate</sender>
   <signal>toggled(bool)</signal>
   <receiver>update_period_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enable_autoupdate</sender>
   <signal>toggled(bool)</signal>
   <receiver>update_period_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
